evennia/web/webclient/static/webclient/js/evennia.js

```diff
@@ -196,62 +214,87 @@ An "emitter" object must have a function
     var WebsocketConnection = function () {
         log("Trying websocket ...");
         var open = false;
```

Comment:
I used 'open' like you use 'ever_open', so I don't think 'open' needs to be there anymore?